examples/blinky/CMakeLists.txt

```diff
@@ -4,9 +4,25 @@ set(CMAKE_TOOLCHAIN_FILE ${CMAKE_CURRENT_SOURCE_DIR}/../../cmake/stm32_gcc.cmake
 project(stm32-blinky C ASM)
 set(CMAKE_INCLUDE_CURRENT_DIR TRUE)
 
-find_package(CMSIS COMPONENTS STM32L0 STM32F1 STM32F4 REQUIRED)
-find_package(HAL COMPONENTS STM32L0 STM32F1 STM32F4 RCC GPIO CORTEX REQUIRED)
-# Find all drivers:
+# F4 example enabled by default
```

Comment:
```suggestion
# Configure here which STM32 target(s) to build
# F4 example enabled by default
```

Reply:
Implemented

Reply:
Using it in #206 seems a good idea and I actually have a F4 board to test this as well. I might supply it in a separate PR

Reply:
@rmspacefish yes please go ahead. This is a clean way to do it.